vud.theme.inc

```diff
@@ -17,7 +17,7 @@ function vud_ctools_plugin_type() {
   return array(
     'widgets' => array(
       'process' => 'vud_width_plugin_defaults',
-        // Themes can offer this plugin.
+      // Themes can offer this plugin.
```

Comment:
Unneeded change, please remove.

Actually, these set of changes should also remove ctools related hook definitions, let's start with `vud_ctools_plugin_type()`, `vud_width_plugin_defaults()`, and `vud_ctools_plugin_directory()`.